src/octree/mod.rs

```diff
@@ -134,57 +139,89 @@ pub struct NodeData {
     pub color: Vec<u8>,
 }
 
+fn bounding_box_to_aabb(bounding_box: &proto::AxisAlignedCuboid) -> Aabb3<f64> {
+    let min = bounding_box.min.clone().unwrap_or_else(|| {
+        let deprecated_min = bounding_box.deprecated_min.clone().unwrap(); // Version 9
+        let mut v = proto::Vector3d::new();
+        v.set_x(f64::from(deprecated_min.x));
+        v.set_y(f64::from(deprecated_min.y));
+        v.set_z(f64::from(deprecated_min.z));
+        v
+    });
+    let max = bounding_box.max.clone().unwrap_or_else(|| {
+        let deprecated_max = bounding_box.deprecated_max.clone().unwrap(); // Version 9
+        let mut v = proto::Vector3d::new();
+        v.set_x(f64::from(deprecated_max.x));
+        v.set_y(f64::from(deprecated_max.y));
+        v.set_z(f64::from(deprecated_max.z));
+        v
+    });
+    Aabb3::new(
+        Point3::new(min.x, min.y, min.z),
+        Point3::new(max.x, max.y, max.z),
+    )
+}
+
 impl Octree {
     // TODO(sirver): This creates an object that is only partially usable.
```

Comment:
Do you know what Holger meant by that? Is that still the case?

Reply:
I do not know as well.. fyi @SirVer, whenever you have time to explain the meaning behind that